packages/dara-core/js/api/websocket.tsx

```diff
@@ -258,48 +182,106 @@ export class WebSocketClient implements WebSocketClientInterface {
 
     closeHandler: () => void;
 
-    constructor(_socket: WebSocket, _token: string, _liveReload = false) {
-        this.socket = _socket;
+    _maxAttempts: number;
```

Comment:
For private variables in JS you can use the [# prefix](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties)